rs/https_outcalls/adapter/src/metrics.rs

```diff
@@ -20,6 +20,12 @@ pub struct AdapterMetrics {
     pub requests: IntCounter,
     /// The number of requests served via a SOCKS proxy.
     pub requests_socks: IntCounter,
+    /// The number of socks connections attempts
+    pub socks_connections_attempts: IntCounterVec,
```

Comment:
```suggestion
    pub socks_connection_attempts: IntCounterVec,
```
But this needs to be propagated everywhere. Feel free to follow my suggestion or not.

Reply:
done